Source/Scene/Expression.js

```diff
@@ -278,6 +323,28 @@ define([
         return this._value;
     };
 
+    Node.prototype._evaluateVariableString = function(feature) {
+        var result = this._value;
+        var regex = /\${(.*?)}/g;
```

Comment:
Construct this just once out of this function as a file-scoped variable.

In general, remember that we want to avoid everything that we can at runtime.

I'm still confident that we can make this a lot faster without the regex, but no need to bother now.